src/components/Quote.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] For best practices and clean code reasons, kindly ensure that you fix all your linters errors from your codebase instead of disabling them.